grapher/controls/EntitySelectorModal.tsx

```diff
@@ -190,7 +196,7 @@ export class EntitySelectorModal extends React.Component<{
                         </h2>
                     </header>
                     <div className="entities wrapper">
-                        <div className="searchResults">
+                        <div className="searchResults" onKeyDown={this.onEscapeKeyDown}>
```

Comment:
We could maybe attach this event handler somewhere higher in the tree to capture events even if the focus element is outside the search results? Maybe the top-level `<div className="entitySelectorOverlay">`

The ideal might be that we even attach it to the `window`... but that's messier, and probably not necessary for this context.

Reply:
I thought about this but didn't know which div would be the best to attach to, thanks for clearing that up! The changes should be pushed now.

Reply:
I think this change might not have made a difference, because it seems `keydown` events are only captured when the focus is on a checkbox or the search input field – sorry about that!